docs/src/route.ts

```diff
@@ -49,6 +49,7 @@ const ROUTES = {
   support: FEATURE_TOGGLE.enable_redirects
     ? '/material-ui/getting-started/support/#professional-support-premium'
     : '/getting-started/support/#professional-support-premium',
+  privacyPolicy: 'https://mui.com/legal/privacy/',
```

Comment:
We are going for this path in https://github.com/mui/mui-store/pull/141.

```suggestion
  privacyPolicy: 'https://mui.com/store/legal/privacy/',
```